code/datums/status_effects/buffs.dm

```diff
@@ -612,6 +612,31 @@
 		H.physiology.burn_mod /=0.8
 		H.physiology.stamina_mod /= 0.8
 
+/datum/status_effect/breaching_and_cleaving
+	id = "breaching_and_cleaving"
+	duration = -1
+	status_type = STATUS_EFFECT_UNIQUE
+	alert_type = /atom/movable/screen/alert/status_effect/breaching_and_cleaving
+
+/atom/movable/screen/alert/status_effect/breaching_and_cleaving
+	name = "Breaching and Cleaving!"
+	desc = "<span class='danger'>Doors, people, machines... nothing will stand before your martial prowess!</span>"
+	icon_state = "breachcleaver"
+
+/datum/status_effect/breaching_and_cleaving/on_apply()
+	. = ..()
+	if(.)
+		if(ishuman(owner))
```

Comment:
if you get a sec, this could be cleaned up either by combining if statements or implementing a guard clause